mockery.yaml

```diff
@@ -0,0 +1,39 @@
+dir: "{{ .InterfaceDir }}/mocks"
+mockname: "{{ .InterfaceName }}"
+outpkg: mocks
+filename: "{{ .InterfaceName | snakecase }}.go"
+packages:
+  github.com/smartcontractkit/chainlink-common/pkg/capabilities/datastreams:
+    interfaces:
+      ReportCodec:
+  github.com/smartcontractkit/chainlink-common/pkg/loop/internal/types:
+    interfaces:
+      Relayer:
+        config:
+          dir: "{{ .InterfaceDir }}/../../mocks"
+  github.com/smartcontractkit/chainlink-common/pkg/monitoring:
+    config:
+      dir: "{{ .InterfaceDir }}"
+      filename: "{{ .InterfaceName | snakecase }}_mock.go"
+      inpackage: true
+      mockname: "{{ .InterfaceName }}Mock"
+    interfaces:
+      Metrics:
+      Source:
+      SourceFactory:
+      Exporter:
+      ExporterFactory:
+  github.com/smartcontractkit/chainlink-common/pkg/services:
+    interfaces:
+      Foo:
```

Comment:
Foo? that not a real thing :) what's going on?

Reply:
CI should be failing :cry:

Reply:
Fixed but I don't understand why it doesn't fail :thinking:

Reply:
why would ci fail? iiuc this change would generate `Foo` mocks that simple aren't used anywhere. why would that additive, unused code fail CI?

Reply:
It is referencing a type which doesn't exist. If we typo something like this, we want it to fail. Otherwise we could accidentally disable things without realizing it.